js/piwik.js

```diff
@@ -4012,6 +4060,7 @@ if (typeof Piwik !== 'object') {
              */
             detectBrowserFeatures();
             updateDomainHash();
+            setVisitorIdCookie();
```

Comment:
Do you remember why this one is needed? It causes cookies to be set even if they might be disabled later via `Tracker.disableCookies`. Can we remove it from the `constructor`?